plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ElasticsearchMetadata.java

```diff
@@ -383,10 +383,12 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
             throw new IllegalArgumentException(format("Unexpected column for table '%s$query': %s", table.getIndex(), column.getName()));
         }
 
-        return ColumnMetadata.builder()
-                .setName(column.getName())
-                .setType(column.getType())
-                .build();
+        return BuiltinColumns.of(column.getName())
+                .map(BuiltinColumns::getMetadata)
```

Comment:
is the change here testable?

Reply:
I'm not sure

Reply:
This PR https://github.com/ssheikin/presto/pull/3 should fix Elasticsearch related tests broken by https://github.com/trinodb/trino/pull/6745

Reply:
https://github.com/ssheikin/presto/pull/3 test for :trino-elasticsearch is green, test for :trino-phoenix is red, as expected.